ner_v2/detectors/numeral/number_range/standard_number_range_detector.py

```diff
@@ -0,0 +1,385 @@
+# coding=utf-8
+import pandas as pd
+import collections
+import os
+import re
+
+from ner_v2.detectors.numeral.constant import NUMBER_RANGE_KEYWORD_FILE_NAME, NUMBER_RANGE_FILE_VARIANTS_COLUMN_NAME, \
+    NUMBER_RANGE_FILE_POSITION_COLUMN_NAME, NUMBER_RANGE_FILE_RANGE_TYPE_COLUMN_NAME, NUMBER_RANGE_MIN_TYPE, \
+    NUMBER_RANGE_MAX_TYPE, NUMBER_RANGE_MIN_MAX_TYPE, NUMBER_REPLACE_TEXT, NUMBER_RANGE_MIN_VALUE, \
+    NUMBER_DETECTION_RETURN_DICT_VALUE, NUMBER_RANGE_MAX_VALUE, NUMBER_RANGE_VALUE_UNIT, \
+    NUMBER_DETECTION_RETURN_DICT_UNIT
+from ner_v2.detectors.numeral.utils import get_list_from_pipe_sep_string
+from ner_v2.detectors.numeral.number.number_detection import NumberDetector
+
+NumberRangeVariant = collections.namedtuple('NumberRangeVariant', ['position', 'range_type'])
+
+
+class BaseNumberRangeDetector(object):
```

Comment:
Once you are done going through most comments. I would like to discuss a potential design change offline, which revolves around handling `entity_value_list`, `original_list`, `processed_text` and `tagged_text`
https://gist.github.com/chiragjn/416070b07b135bf3764ef366691cc36d